molo/profiles/admin.py

```diff
@@ -110,3 +127,116 @@ class FrontendUsersModelAdmin(WagtailModelAdmin, ProfileUserAdmin):
 
     def get_queryset(self, request):
         return User.objects.filter(profile__site=request.site)
+
+
+class TzDateTimeWidget(DateTimeWidget):
+
+    def render(self, value, obj):
```

Comment:
This is not being tested and is showing up in coveralls -> could we add a test here please?